src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -208,7 +219,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
    * First queries for the existence of the auth file, then deletes it if it exists.
    * If either of these operations fails, then a Future.failure is returned
    */
-  override def cleanUpForWorkflow(workflow: WorkflowDescriptor)(implicit ec: ExecutionContext): Future[Unit] = authenticated { connection =>
+  override def cleanUpForWorkflow(workflow: WorkflowDescriptor)(implicit ec: ExecutionContext): Future[Unit] = authenticatedAsCromwell { connection =>
```

Comment:
This method could maybe have a better name, it almost sounds like it's doing compaction rather than deleting the auth file.